src/file.cr

```diff
@@ -11,13 +11,23 @@ require "crystal/system/file"
 # content = file.gets_to_end
 # file.close
 #
-# # Implicit close with `open`
+# # Implicit close with `open` and a block:
 # content = File.open("path/to/file") do |file|
 #   file.gets_to_end
 # end
 #
-# # Shortcut:
+# # Shortcut of the above:
 # content = File.read("path/to/file")
+#
+# # Write to a file by opening with a "write mode" specified.
+# File.open("path/to/file", "w") do |file|
+#   file.print "hello"
+# end
+# # Content of file on disk will now be "hello".
+# See `self.new` for various options *mode* can be.
```

Comment:
```suggestion
# # See `.new` for various options *mode* can be.
```

Reply:
OK for some reason it wasn't linkable when within a code block either way, so I moved it out and adopted this in feeac26c8, the link works now, thanks for the suggestion.